### PR TITLE
Preview 2 update

### PR DESCRIPTION
**Pull request by Ercenk**
Preview 2 with AutoRest C# generator V3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,56 @@
-![.NET Core](https://github.com/Azure/commercial-marketplace-client-dotnet/workflows/.NET%20Core/badge.svg)
+![.NET Core](https://github.com/Azure/commercial-marketplace-saas-sdk-client-dotnet/workflows/.NET%20Core/badge.svg)
```
Comment (scseely) [inline README.md:1]: Location should be github.com/microsoft, not github.com/Azure

File: src/Microsoft.Marketplace.Core/Constants.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Marketplace.Core
+{
+    public static class Constants
+    {
+        public const string DefaultMarketplaceScope = "20e940b3-4c77-4b0b-9a53-9e16a1b010a7/.default";
```
Comment (scseely) [inline src/Microsoft.Marketplace.Core/Constants.cs:8]: In the Python client, this "fact" can be set by using the autorest parameter 
`--credential-scopes=20e940b3-4c77-4b0b-9a53-9e16a1b010a7/.default`

Can you check if the same is true for C#? If so, this constant and file might be able to go away.

File: src/Microsoft.Marketplace.Metering/Configuration.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"OutputFolder":".","Namespace":"Microsoft.Marketplace.Metering","LibraryName":"Metering","SharedSourceFolders":["C:\\Users\\ercenk\\.nuget\\packages\\autorest.csharp.v3\\3.0.0-beta.20201217.1\\content\\Azure.Core.Shared\\","C:\\Users\\ercenk\\.nuget\\packages\\autorest.csharp.v3\\3.0.0-beta.20201217.1\\content\\Generator.Shared\\"],"AzureArm":false,"PublicClients":true,"ModelNamespace":true,"HeadAsBoolean":false}
```
Comment (scseely) [inline src/Microsoft.Marketplace.Metering/Configuration.json:1]: This file includes a path to your local machine: c:\users\ercenk\...
Consider removing the file or making it more 'generic'.

File: src/Microsoft.Marketplace.SaaS/Configuration.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"OutputFolder":".","Namespace":"Microsoft.Marketplace.SaaS","LibraryName":"SaaS","SharedSourceFolders":["C:\\Users\\ercenk\\.nuget\\packages\\autorest.csharp.v3\\3.0.0-beta.20201217.1\\content\\Azure.Core.Shared\\","C:\\Users\\ercenk\\.nuget\\packages\\autorest.csharp.v3\\3.0.0-beta.20201217.1\\content\\Generator.Shared\\"],"AzureArm":false,"PublicClients":true,"ModelNamespace":true,"HeadAsBoolean":false}
```
Comment (scseely) [inline src/Microsoft.Marketplace.SaaS/Configuration.json:1]: Same as before: this is a machine dependent setup-- will fail for almost everyone but you.

File: tests/Azure.Core.TestFramework/Azure.Core.TestFramework.sln [hunk 5]
```diff
@@ -0,0 +1,25 @@
+﻿
```
Comment (scseely) [inline tests/Azure.Core.TestFramework/Azure.Core.TestFramework.sln:1]: Is the only way to get this code to copy + paste it?

**Review by scseely — CHANGES_REQUESTED** [review]

File: tests/Azure.Core.TestFramework/Azure.Core.TestFramework.sln (hunk 5 above)
Comment (Ercenk) [reply tests/Azure.Core.TestFramework/Azure.Core.TestFramework.sln:1]: Yes

**Review by scseely — APPROVED** [review]

**Pull request merged**